.gitpod.yml

```diff
@@ -0,0 +1,47 @@
+tasks:
+  - init: |
+      npm i &&
+      npm run dx:up &&
+      cp .env.example .env &&
+      set -a; source .env &&
+      sed -i -e "s|http://localhost:3000|$(gp url 3000)|" .env \
+             -e "s|http://localhost:3001|$(gp url 3001)|" .env \
+             -e "s|http://localhost:9000|$(gp url 9000)|" .env 
```

Comment:
Just use `export NEXT_AUTH="$(gp url 3000)"` for these items rather than `sed` please

Reply:
okk

Reply:
Done